debian/control

```diff
@@ -0,0 +1,46 @@
+Source: seafile-gui
+Section: net
+Priority: extra
+Maintainer: m.eik michalke <meik.michalke@hhu.de>
+Build-Depends:
+    debhelper (>= 7),
+    dh-python,
+    python,
```

Comment:
`Seafile-client` has no python modules, so I guess we can remove `dh-python` and `python` from the build deps?